.devcontainer/Dockerfile

```diff
@@ -10,7 +10,10 @@ ARG USER_GID=$USER_UID
 RUN if [ "$USER_GID" != "1000" ] || [ "$USER_UID" != "1000" ]; then groupmod --gid $USER_GID vscode && usermod --uid $USER_UID --gid $USER_GID vscode; fi
 
 RUN apt update && export DEBIAN_FRONTEND=noninteractive \
-    && apt -y install postgresql-client python3-pip python3-distutils jq zip unzip git bash-completion
+    && apt -y install postgresql-client python3-pip python3-distutils jq zip unzip git bash-completion locales
+
+ENV LANG en_US.UTF-8  
+ENV LANGUAGE en_US:en 
```

Comment:
I'm having issues running stuff in the new container without this. Haven't seen this with other containers using the same base image so it's a bit odd to me but this resolves it